packages/trader/src/Modules/Trading/Components/Form/Purchase/contract-info.tsx

```diff
@@ -150,18 +153,16 @@ const ContractInfo = ({
                     )
                 )}
             </div>
-            {!is_multiplier && !is_accumulator && (
-                <DesktopWrapper>
-                    <Popover
-                        alignment='left'
-                        icon='info'
-                        id={`dt_purchase_${type.toLowerCase()}_info`}
-                        is_bubble_hover_enabled
-                        margin={216}
-                        message={has_error_or_not_loaded ? '' : setHintMessage()}
-                        relative_render
-                    />
-                </DesktopWrapper>
+            {!is_multiplier && !is_accumulator && !isMobile && (
```

Comment:
In this file you didn't remove `DesktopWrapper` and `MobilepWrapper`. Are we gonna use them? I can see them in the code above